src/components/SwipeableView/index.ios.js

```diff
@@ -0,0 +1,40 @@
+import React, {PureComponent} from 'react';
+import {PanResponder, View} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    children: PropTypes.element.isRequired,
+
+    // callback whenever swipe down is registered
+    onSwipeDown: PropTypes.func.isRequired,
+};
+
+class SwipeableView extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.panResponder = PanResponder.create({
+
+            // The PanResponder gets focus only when the y-axis movement is over 3 pixels
+            onMoveShouldSetPanResponderCapture:
+            (_, gestureState) => gestureState.dy > 3,
```

Comment:
If we are not using this event we can do `_event` instead of `_` to avoid confusion with the use of `underscore` elsewhere.

Reply:
Yup, makes sense!